Peekabook/Peekabook/Presentation/UIComponents/CustomTextView.swift

```diff
@@ -115,7 +114,7 @@ extension CustomTextView {
 
 extension CustomTextView: UITextViewDelegate {
     func textViewDidBeginEditing(_ textView: UITextView) {
-        if (textView.text == I18N.BookDetail.commentPlaceholder) ||
+        if (textView.text == I18N.BookDetail.commentPlaceholder + "           ") ||
```

Comment:
p3;
위에 프로퍼티로 만들어주세용 (하드코딩 x)
```swift
private let placeholderBlank: String = "          "
```

```suggestion
        if (textView.text == I18N.BookDetail.commentPlaceholder + placeholderBlank) ||
```

Reply:
배워갑니닷!!! 👍🏻